client/jetpack-cloud/sections/partner-portal/revoke-license-dialog/index.tsx

```diff
@@ -94,7 +94,7 @@ export default function RevokeLicenseDialog( {
 
 			<p>
 				{ translate(
-					'A revoked license cannot be reused and the associated site will no longer have access to the provisioned product. You will stop being billing for this license immediately.'
+					'A revoked license cannot be reused, and the associated site will no longer have access to the provisioned product. You will stop being billed for this license immediately.'
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.